test/integration/save_world.cc

```diff
@@ -855,6 +857,104 @@ TEST_F(SdfGeneratorFixture, WorldWithLights)
   ASSERT_NE(nullptr, genWorld);
 }
 
+/////////////////////////////////////////////////
+TEST_F(SdfGeneratorFixture, ModelWithJoints)
+{
+  this->LoadWorld("test/worlds/joint_sensor.sdf");
```

Comment:
ignition::common::JoinPaths

```suggestion
  this->LoadWorld("test/worlds/joint_sensor.sdf");
```

Reply:
done. ceafa41